### PR TITLE
Projects/brewfind python webapp

### PR DESCRIPTION
**Pull request by Rmariner25**
Adding BrewFind under projects section

### CODE REVIEW

File: src/routes/projects/+page.svx [hunk 1]
```diff
@@ -73,6 +73,10 @@ An Open Brewery DB GraphQL server implementation in .NET Core and maintained by
 
 A Python interface for the API built and maintained by [@jrbourbeau](https://github.com/jrbourbeau/)
 
+#### [BrewFind Django App](https://github.com/Rmariner25/moengage_brewery)
+
+A Django-based web application to search, add review, and view reviews for breweries. Implemented search autocomplete feature among other features such as caching api results, pagination, embeded Google Maps for breweries. [@jrbourbeau](https://github.com/Rmariner25)
```
Comment (chrisjm) [inline src/routes/projects/+page.svx:78]: `[@jrbourbeau](https://github.com/Rmariner25)` should probably be `[@Rmariner25](https://github.com/Rmariner25)` :)

**Review by chrisjm — CHANGES_REQUESTED** [review]
Thanks for adding @Rmariner25 ! Just one correction I saw and I'll merge it!

File: src/routes/projects/+page.svx (hunk 1 above)
Comment (Rmariner25) [reply src/routes/projects/+page.svx:78]: Sorry for the oversight. Thanks for the merge.

**Comment by Rmariner25** [comment]
@chrisjm I have made the required changes. Please do the needful.

**Review by chrisjm — APPROVED** [review]
Awesome! Thanks so much @Rmariner25 ! 🍻

**Pull request merged**